### PR TITLE
Configurator docker start fix

### PR DESCRIPTION
**Pull request by dpaks**
@ashutosh-mishra @ajayanto 

Verified that configurator docker starts alongside other dockers when Visibility VM boots up.

### CODE REVIEW

File: devstack/exercises/nfp_service/user-data/visibility_user_data [hunk 1]
```diff
@@ -76,3 +76,4 @@ write_files:
 
 runcmd:
   - cd /usr/share/visibility/docker/ && bash rundocker.sh
+  - docker run -d --restart=always --name configurator -it -p 5672:5672 -p 8070:8080 configurator-docker
```
Comment (ajayanto) [inline devstack/exercises/nfp_service/user-data/visibility_user_data:79]: For preventing access from visibility external network  please add   --net=internalbr to the runcmd, 
 docker run -d  --restart=always  --name configurator -it --net=internalbr -p 5672:5672 -p 8070:8080 configurator-docker

**Comment by ashutosh-mishra** [comment]
Do we need this change now, @hrishikeshtak mentioned that visibility and configurator vms are different now.

**Comment by ashutosh-mishra** [comment]
@bhyrava @aravinda64

**Comment by dpaks** [comment]
At present, NFP enterprise installation is broken with configurator docker failing to start. So, this can go as a temporary fix until we have the '2 VMs patch' in NFP. @ashutosh-mishra

**Comment by njagadish** [comment]
Not required. Fixed the code to work with two VMs for enterprise mode.

**Pull request closed without merging**